GX-AnalyticContinuation/ABINIT/m_simple_pade.f90

```diff
@@ -0,0 +1,205 @@
+module m_simple_pade
+
+  implicit none
+
+  public :: pade
+
+  public
+  integer, parameter :: dp=kind(1.0d0)
+  integer, parameter :: dpc=kind((1.0_dp,1.0_dp))
+  real(dp), parameter :: zero=0._dp
+  complex(dpc), parameter :: czero = (0._dp,0._dp)
+  complex(dpc), parameter :: cone  = (1._dp,0._dp)
+
+
+
+contains
+
+!----------------------------------------------------------------------
+
+!!****f* m_numeric_tools/pade
+!! NAME
+!!  pade
+!!
+!! FUNCTION
+!!  Calculate the pade approximant in zz of the function f calculated at the n points z
+!!
+!! SOURCE
+
+function pade(n,z,f,zz)
+
+!Arguments ------------------------------------
+!scalars
```

Comment:
This sort of documenting is not helpful (but I see it a lot in other packages). I can read from the declarations what type the data is. But what is n, zz, etc? 

Document what variables are